src/enzyme_ad/jax/Utils.cpp

```diff
@@ -660,13 +660,22 @@ SymmetricResultAnalysis::State SymmetricResultAnalysis::localGuaranteed(
     }
   }
 
+  if (isa<enzymexla::SymmOp>(op)) {
+    return State::GUARANTEED;
+  }
```

Comment:
Is this generally true? it is true for syrk but not for symm if B is not symmetric

Reply:
yup you're right, there's no op for `syrk` yet right?